src/index.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.14.0/css/all.css">
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0-alpha1/dist/css/bootstrap.min.css" rel="stylesheet"
+        integrity="sha384-GLhlTQ8iRABdZLl6O3oVMWSktQOp6b7In1Zl3/Jr59b6EGGoI1aFkw7cmDA6j6gD" crossorigin="anonymous">
+
+
+    <title>Wbpack Exercise</title>
+</head>
+
+<body>
+
+
+
+    <main class=" container-fluid">
+
+        <!----------------todo Lists header ----------------->
+        <div class="row">
+            <div class="col-12" id="todays">
+                <h3>Today's To Do</h3>
+                <i class="fa fa-sync"></i>
+
+            </div>
+
+        </div>
+        <!----------------todo Lists input ----------------->
+        <div class="row" type="input" id="input">
+            <form >
+            <div class="input-group ">
+                <div class="input-group-prepend">
+                </div>
+                <input type="text" class="form-control col" placeholder="Add Your list..." id="lists-input">
+                <button class="input-group-text" type="submit" id="litsBtn">
+                    <i class="fas fa-reply"  ></i>
+                </button>
+            </div>
+        </form>
+
+
+
+        </div>
+        <!----------------todo Lists ----------------->
+
+        <div class="row" id="list-holder">
+            
+        </div>
+
+
+        <!----------------todo Lists  footer----------------->
+        <div class="row">
+            <div class="col-12" id="footer">
+                <p>clear all completed</p>
+            </div>
+        </div>
```

Comment:
- [ ] Great effort working on your project the TodoList structure, 👍 👍  but currently, the clear all complete functionality after the user has completed his todo tasks is not working at the moment. As shown in this [video](https://www.youtube.com/watch?v=AcUd-_Yjjqg) the clear all completed is working after the user has completed the necessary to-do tasks.

- Kindly work on it.